scripts/create-update.ps1

```diff
@@ -0,0 +1,92 @@
+<#
+Copyright (c) AppDynamics, Inc., and its affiliates 2015
+All rights reserved
+
+Build script for wpt-driver.
+
+REQUIREMENTS:
+- msbuild.exe and python.exe (required by the Chrome compile.cmd) should be in
+  the path
+
+This script has been tested in GitHub's Git Shell.
+Executing it in a regular PowerShell may also work, but you will have to
+setup the path for each dependencies.
+#>
+
+param(
+    [string]$configuration = "Release"
```

Comment:
Does this provide a default command-line param of Release? If so, what other params could we expect to see?